aii-opennebula/src/main/pan/quattor/aii/opennebula/schema.pan

```diff
@@ -86,7 +88,23 @@ type opennebula_vmtemplate_datastore = string{} with {
     return(true);
 };
 
+@documentation{ 
+check if the network interface is available from the quattor tree
+}
+function is_valid_interface_ignoremac = {
+    foreach (idx; attr; SELF) {
+        if (! exists("/system/network/interfaces/"+attr)) {
+            error(format("ignoremacinterface: '%s' is not available from /system/network/interfaces tree", attr));
+            return(false);
+        };
+    };
+    return(true);
+};
+
+
 type opennebula_vmtemplate = {
     "vnet" : opennebula_vmtemplate_vnet
     "datastore" : opennebula_vmtemplate_datastore
+    "ignoremacaddr" ? type_hwaddr[] # MAC address will be ignored for these mac addr type_hwaddr
```

Comment:
use a new type `opennebula_ignoremac` which has attributes `addr` and/or `interface`. it's cleaner then common prefixes in attribute names